packages/mux-active-viewer-count/src/index.ts

```diff
@@ -0,0 +1,225 @@
+import { globalThis } from 'shared-polyfills';
+
+export const subscribeViewerCount = (
+  token: string,
+  pollInterval: number,
+  callback: (views: number) => void,
+  errorCb: (errorMsg: string) => void
+) => {
+  const url = `https://stats.mux.com/counts?token=${token}`;
+  const controller = new AbortController();
+  const signal = controller.signal;
+  let timeoutId: number | undefined;
+  let aborted = false;
```

Comment:
Theoretically, this could be simplified to rely on `signal.aborted` instead of having a separate variable tracking `aborted` here.

Reply:
@gkatsev Given the "async recursion" and the location of the `errorCb` usage, I didn't see a straightforward way to not include the `aborted` flag, but I might not have been creative enough. If you see a clean refactor that won't lead to other weirdness, I'd definitely prefer to remove the flag.

Reply:
it's definitely not a blocker, but `signal` is in scope here, so, it seems to be as simple as `s/\baborted\b/signal.aborted/`, but I haven't tried it.

Reply:
Ah I see so you're proposing still relying on a flag but just using `signal.aborted` as the flag. I thought (hoped?) you saw a way to get rid of its need altogether, but your proposal should work. @jaredsmith mind refactoring and confirming that it still works? (can help with this if needed)